src/main/java/org/sagebionetworks/web/client/jsinterop/mui/Grid.java

```diff
@@ -0,0 +1,91 @@
+package org.sagebionetworks.web.client.jsinterop.mui;
+
+import org.sagebionetworks.web.client.jsinterop.ReactComponentType;
+import org.sagebionetworks.web.client.jsinterop.react.HasStyle;
+
+public class Grid extends HasStyle<ReactComponentType<GridProps>, GridProps> {
+
+  public Grid() {
+    super(MaterialUI.Unstable_Grid2, GridProps.create(false));
+  }
+
+  public void setId(String id) {
+    props.id = id;
+    this.render();
+  }
```

Comment:
When a class that extends `ReactComponentV2` needs to update props, you can just update `this.props` (which matches the generic type) and then call `this.render()`